### ISSUE TITLE
Add support for UM431DA

### ISSUE THREAD

**Issue by mmatuska** [dominiksalvet/asus-fan-control#71]
**Description**
I would like to help adding support for the UM431DA

**Reason**
I want to start my fan at a higher temperature.

**Additional context**
acpi_call on kernel 5.5 or 5.6 (https://github.com/vikmik/acpi_call) returns AE_NOT_FOUND when running afc-scout

**Comment by dominiksalvet** [comment]
Transferred once again since the AE_NOT_FOUND has become a notorious problem of ASUS devices since the creation of this issue. Here we can group those issues under the same label.

**Comment by cemkaya-mpi** [comment]
Is there a working fan solution? I also have this model and the same issue. The fan control does not work.

**Comment by mmatuska** [comment]
I got used to the fan noise in the meantime :1st_place_medal:

**Comment by cemkaya-mpi** [comment]
Its driving me nuts. The fan goes crazy at 55°C. I underclocked the CPU to 1.7 GHz so that it stays about 45-50°C :(

**Comment by mmatuska** [comment]
Do you have the latest BIOS?

**Comment by cemkaya-mpi** [comment]
I'm on version 304.

**Comment by dominiksalvet** [comment]
This laptop has a Ryzen processor, right? Have you tried https://github.com/FlyGoat/RyzenAdj by chance?

**Comment by cemkaya-mpi** [comment]
Yes, it has a Ryzen procesor, and I have tried it. Not everything works, but I was able bring the CPU power usage down using  `ryzenadj --fast-limit=12000 --slow-limit=12000 --stapm-limit=12000` This results in less fan volume, but the cpu is throttled at 12Watts which gives minimum Performance. As far as I can tell, the temperature limit I set with `--tctl-temp` has no effect on fan speed. I could not find a setting to directly control the fan speed.
